src/group.coffee

```diff
@@ -219,18 +216,22 @@ module.exports = (robot) ->
       return
     response = []
     for name in names
-      if group.remove g, name
-        response.push "#{name} removed from group #{g}."
-      else
-        response.push "#{name} is not in group #{g}!"
+      for k, u of robot.brain.data.users
```

Comment:
Maybe add a case that gives an error message when the name you put in doesn't match any of the names?